ts/editor/FieldState.svelte

```diff
@@ -12,8 +12,26 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         justify-content: flex;
         flex-grow: 1;
 
-        & > :global(*) {
-            margin-left: 2px;
+        /* replace with "gap: 6px" as soon as possible */
```

Comment:
Might be worth mentioning in the comment that it's blocked on an update to Chromium 84 (Qt6 only) and iOS 14.1 Same in FieldsEditor